marklogic-data-hub/src/main/java/com/marklogic/hub/dataservices/ProvenanceService.java

```diff
@@ -56,7 +56,7 @@ private ProvenanceServiceImpl(DatabaseClient dbClient, JSONWriteHandle servDecl)
                 this.baseProxy = new BaseProxy("/data-hub/5/data-services/provenance/", servDecl);
 
                 this.req_pruneProvenance = this.baseProxy.request(
-                    "pruneProvenance.sjs", BaseProxy.ParameterValuesKind.MULTIPLE_NODES);
+                    "deleteProvenance.sjs", BaseProxy.ParameterValuesKind.MULTIPLE_NODES);
```

Comment:
Run "gradle generateProvenanceInterface", that will update this file, which should include updating the pruneProvenance references too.